src/components/input/UIInput.vue

```diff
@@ -3,7 +3,7 @@
         <UIExpandTransition>
             <label v-if="label || $slots.label"
                    :for="$attrs.id ?? name"
-                   class="font-medium text-color inline-flex mb-1.5"
+                   class="font-medium text-color inline-flex items-center mb-1.5"
```

Comment:
Can't figure this one out mate - what does this solve?

Reply:
When I customised the label like
```html
<UIInput>
  <template #label>
    My Label <Icon name="my-icon" />
  <template>
</UIInput />
```
The label and icon would not be centered. With this change, they will be.